apps/ml/site/analysis/analysis.jag

```diff
@@ -403,8 +403,8 @@
                     handleNotification(successText, '#notification-area', 'success');
                 },
                 error : function(res){
-                var errorText = res.responseText
-                    handleNotification(errorText, '#notification-area', 'warning');
+                var errorText = JSON.parse(res.responseText)["exception"];
```

Comment:
I've tried both ways.
AFAIK, this was the only way that I could actually get the actual exception without the tag `exception` and empty responses.